package.json

```diff
@@ -47,16 +47,16 @@
     "winston": "^2.4.0"
   },
   "devDependencies": {
-    "@types/expect": "^1.20.3",
     "@types/body-parser": "^1.16.8",
+    "@types/expect": "^1.20.3",
     "@types/express": "^4.11.0",
     "@types/jsonwebtoken": "^7.2.5",
     "@types/redlock": "^3.0.1",
     "@types/sequelize": "^4.27.4",
     "@types/winston": "^2.3.7",
     "npm-run-all": "^4.0.1",
     "rimraf": "2.6.1",
-    "sqlite3": "^4.0.2",
+    "sqlite3": "^4.0.4",
```

Comment:
All dependency changes need to be in their own commit (With a clear commit summary), reason being that if we have a production crisis and need to rollback we now what commit does what.